tests/test_main.py

```diff
@@ -0,0 +1,25 @@
+from typer.testing import CliRunner
+
+runner = CliRunner()
+
+
+def test_gatorgrade_runs_correct():
+    """Test that ensures that the default command runs correctly."""
+    assert True
+
+
+def test_generate_creates_correctly():
+    """Test that ensures that the generate command creates the .yml file correctly."""
+    assert True
+
+
+def test_generate_fail_with_existing_yml():
```

Comment:
This test name works fairly well with the convention outlined above; however `fail` probably should be plural.